mediapipe/tasks/ios/vision/hand_landmarker/sources/MPPHandLandmarksConnections.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import "mediapipe/tasks/ios/components/containers/sources/MPPConnection.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+NSArray<MPPConnection *> *const kHandPalmConnections = @[
```

Comment:
"Prefixes are commonly required in Objective-C to avoid naming collisions in a global namespace. Classes, protocols, global functions, and global constants should generally be named with a prefix that begins with a capital letter followed by one or more capital letters or numbers."

vs

"A lowercase k can be used as a standalone prefix for constants of static storage duration declared within implementation files:
NOTE: Previous convention was for public constant names to begin with a lowercase k followed by a project-specific [prefix](https://g3doc.corp.google.com/company/teams/objc-readability/style_guide.md?cl=head#prefixes). This practice is no longer recommended."

These are pretty close to being global constants, so we should probably follow this guideline.